packages/marshal/src/helpers/passStyle-helpers.js

```diff
@@ -13,6 +13,10 @@ const {
 const { apply } = Reflect;
 const { isArray } = Array;
 
+/** @type {Checker} */
+export const conditionChecker = (cond, _details = undefined) => cond;
```

Comment:
I don't love the name `conditionChecker` but I don't have a better suggestion. I do see why you needed to introduce something.

Reply:
I'm not clear on why we all the `check = conditionChecker` can't be simply `check = v => v`. Is it because we need that default function to be hardened?

If so, wouldn't it still be better to name this function by what it does instead of how it's used (as yet)? E.g. `identity`. I imagine it would be useful to have a hardened identity function as a general utility.

Reply:
I believe that's because of the way type inference works now if no explicit `@param` have been defined for `check`. If you just use a `check = v => v`, then the type of check will be inferred as `(v: any) => any`, and thus will only have 1 allowed arg instead of 2.

Reply:
The type is defined by `PassStyleHelper` so it's not a type problem. Maybe it was before and something improved since then? 

If there's no runtime need for the `conditionChecker` function then I'd suggest including 9aba0c028bd501d943b8b83c1ec0e4d0d6d6e05d in this  PR

Reply:
Reverted as I found no need for this anymore.